docs/docs/integrations/tools/databricks.ipynb

```diff
@@ -34,11 +34,19 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 1,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Note: you may need to restart the kernel to use updated packages.\n"
+     ]
+    }
+   ],
    "source": [
-    "%pip install --upgrade --quiet databricks-sdk langchain-community mlflow"
+    "%pip install --upgrade --quiet databricks-sdk langchain-community langgraph mlflow"
    ]
   },
   {
```

Comment:
nit: Can we the update import for `ChatDatabricks` to `langchain-databricks`?

Reply:
Good call, done!